templates/dev_files.go

```diff
@@ -0,0 +1,83 @@
+package templates
+
+var devFilesTemplate = projectTemplate{
+	Dirs:  getCartridgeDevDirs(),
+	Files: getCartridgeDevFiles(),
+}
+
+func getCartridgeDevDirs() []dirTemplate {
+	return []dirTemplate{
+		dirTemplate{
+			Path: "tmp",
+			Mode: 0755,
+		},
+	}
+}
+
+func getCartridgeDevFiles() []fileTemplate {
+	depsScript := fileTemplate{
+		Path: "deps.sh",
+		Mode: 0755,
+		Content: `#!/bin/sh
```

Comment:
I propose to stop inline such texts and move it into variables (and may be files).